src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -79,5 +86,147 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
         }
 
         #endregion
+
+        #region BulletSupport
+
+        /// <summary>
+        /// Method used to change the text property of the note to support bullet points.
+        /// It first personalizes the TAB spacing by converting it to space indentation. (The default spacing is too wide)
+        /// It then converts dashes to bullet characters depending on their existing indentation.
+        /// It then corrects bullet characters if their indentation has changed 
+        /// </summary>
+        /// <param name="text"> Input text with no bullet support </param>
+        /// <returns> Bullet supported text </returns>
+        private string AddBulletPointSupport(string text)
+        {
+            if (text.Length == 0)
+                return text;
+
+            // Definition of spacing used in indentation
+            string firstTabIndent = LEFT_INDENT + TAB_INDENT;
+            string secondTabIndent = LEFT_INDENT + TAB_INDENT + TAB_INDENT;
+            
+            // Characters and indentation used for the 3 bullet types
+            string leftIndentedBullet = LEFT_INDENT + ROUND_FILLED_BULLET + RIGHT_INDENT;
+            string firstTabIndentedBullet = firstTabIndent + ROUND_EMPTY_BULLET + RIGHT_INDENT;
+            string secondTabIndentedBullet = secondTabIndent + SQUARE_FILLED_BULLET + RIGHT_INDENT;
+
+            // Convert TAB input to the appropiate bullet and indentation
+            text = TurnTabsToSpaces(
```

Comment:
Do we need to do this every time the text is being edited? Couldn't we implement some checks?

```csharp
if(text.Contains(tabChar))
{
    do something....
}

... other checks
```